.ci/openshift-ci/build/Dockerfile

```diff
@@ -24,17 +24,17 @@ ADD . /che-plugin-registry
 WORKDIR /che-plugin-registry
 
 # Install build tools
-RUN yum update -y && yum upgrade -y
-RUN yum install git -y 
-RUN yum install openssh -y
-RUN curl -sL https://rpm.nodesource.com/setup_14.x | bash -
-RUN yum install nodejs -y
-RUN npm install --global yarn
+RUN yum update -y && yum upgrade -y &&\
+    yum install git -y &&\
+    yum install openssh -y &&\
+    curl -sL https://rpm.nodesource.com/setup_14.x | bash - &&\
+    yum install nodejs -y &&\
+    npm install --global yarn
 
 # Build repo
-RUN yarn && \
-    yarn --cwd tools/build build && \
-    eval node tools/build/lib/entrypoint.js --output-folder:"output"
+RUN yarn && yarn install
```

Comment:
Wouldn't it be easier to run the `build.sh` script instead of the commands directly? You can run it so that it doesn't produce a build, i.e. `build.sh --skip-oci-image`

Reply:
Good catch, thank you. Done.